mace/python/tools/converter_tool/transformer.py

```diff
@@ -1163,7 +1172,7 @@ def transpose_filters(self):
 
             for op in net.op:
                 if (op.type == MaceOp.Conv2D.name or
-                    op.type == MaceOp.Deconv2D.name) and\
+                    op.type == MaceOp.DepthwiseConv2d.name) and\
```

Comment:
DepthwiseConv2d of CPU quantize does not need to transpose.

Reply:
fixed in commit 689b236